thin-backend-react/auth-ui.tsx

```diff
@@ -267,14 +271,14 @@ function SignUpForm({ description, onLoginClick }) {
         <div>
             <button type="submit" disabled={isLoading}>
                 {isLoading
-                    ? <span className="thin-login-spinner-border thin-login-spinner-border-sm" role="status" aria-hidden="true"></span>
+                    ? <LoadingSpinner small/>
                     : "Sign Up"
                 }
             </button>
         </div>
 
         <p>
-            <span className="thin-login-signup">Already have an account?</span> <a href={ihpBackendUrl('/NewUser')} onClick={() => { event.preventDefault(); onLoginClick(); }}>Login</a>
+            <span className="thin-auth-signup">Already have an account?</span> <a href={ihpBackendUrl('/NewUser')} onClick={() => { event.preventDefault(); onLoginClick(); }}>Login</a>
```

Comment:
Will this work without JS? That's pretty neat.

Reply:
Basically yes, but if you have no JS enabled, the app might not correctly handle the redirect back from the thin login page (Thin redirects back to the SPA url and passes `userId` and a random string `accessToken` via query parameters. These then need to be exchanged for a JWT via an HTTP call)